todoman/ui.py

```diff
@@ -210,67 +209,62 @@ def __init__(self, date_format, time_format, dt_separator):
         self.now = datetime.datetime.now().replace(tzinfo=self._localtimezone)
         self.tomorrow = self.now.date() + datetime.timedelta(days=1)
 
-        # An empty date which should be used in case no date is present
-        self.date_width = len(self.now.strftime(date_format))
-        self.empty_date = " " * self.date_width
-
-        self.time_width = len(self.now.strftime(time_format))
-        self.empty_time = " " * self.time_width
         # Map special dates to the special string we need to return
         self.special_dates = {
-            self.now.date(): "Today".rjust(self.date_width, " "),
-            self.tomorrow: "Tomorrow".rjust(self.date_width, " "),
+            self.now.date(): "Today",
+            self.tomorrow: "Tomorrow",
         }
         self._parsedatetime_calendar = parsedatetime.Calendar()
 
     def compact(self, todo):
-        """
-        Returns a brief representation of a task, suitable for displaying
-        on-per-line.
-
-        :param Todo todo: The todo component.
-        """
-        # completed = "✓" if todo.percent_complete == 100 else " "
-        completed = "X" if todo.is_completed else " "
-        percent = todo.percent_complete or ''
-        if percent:
-            percent = " ({}%)".format(percent)
-        urgent = " " if todo.priority in [None, 0] else "!"
-
-        due = self.format_datetime(todo.due)
-        if todo.due and todo.due <= self.now and not todo.is_completed:
-            due = click.style(due, fg='red')
-
-        summary = todo.summary
-        list = self.format_database(todo.list)
-
-        return self.compact_format.format(
-            completed=completed,
-            due=due,
-            id=todo.id,
-            list=list,
-            percent=percent,
-            summary=summary,
-            urgent=urgent,
-        )
+        return self.compact_multiple([todo])
+
+    def compact_multiple(self, todos):
+        table = []
+        for todo in todos:
+            completed = "X" if todo.is_completed else " "
+            percent = todo.percent_complete or ''
+            if percent:
+                percent = " ({}%)".format(percent)
+            urgent = " " if todo.priority in [None, 0] else "!"
+
+            due = self.format_datetime(todo.due)
+            if todo.due and todo.due <= self.now and not todo.is_completed:
+                due = click.style(due, fg='red')
+
+            table.append([
+                "{:3d}".format(todo.id),
```

Comment:
Removed the formating. This should be a plus to you, since you have one-digit tasks. :D